cqrs-spring-utils/build.gradle

```
@@ -0,0 +1,6 @@
+dependencies {
+
+    implementation project(":bus-cqrs-api")
+
+    compile "org.springframework:spring-context:4.3.14.RELEASE"
```

Should be `api`, the types from `spring-context` are exposed to consumers of this module right?
```suggestion
    api "org.springframework:spring-context:4.3.14.RELEASE"
```